sys/vm/vm_fault.c

```diff
@@ -1314,12 +1314,25 @@ vm_fault_cow(struct faultstate *fs)
 		 * The destination page will always belong to a
 		 * tag-bearing VM object.
 		 */
-		KASSERT(fs->first_object->flags & (OBJ_HASCAP | OBJ_NOCAP),
-		    ("%s: destination object %p doesn't have OBJ_HASCAP",
+		KASSERT((fs->first_object->flags & (OBJ_HASCAP | OBJ_NOCAP)) &&
```

Comment:
The second paragraph in the commit log seems to end abruptly for this change?

```
In particular, the background
scan must visit any page that has PGA_CAPSTORE set, and the CLG.
```

Reply:
Oops, thanks. Fixed now.